tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -185,6 +185,33 @@ def spectre_meltdown_reported_vulnerablities(
     }
 
 
+def check_vulnerabilities_on_guest(status):
+    """
+    There is a REPTAR exception reported on m6i.metal for
+    guest kernel versions 4.14/5.10/6.1 when spectre-meltdown-checker.sh
+    script is run inside the guest from below the tests:
+        test_spectre_meltdown_checker_on_guest and
+        test_spectre_meltdown_checker_on_restored_guest
+    The same script when run on host doesn't report the
+    exception which means the instances are actually not vulnerable to REPTAR.
+    The only reason why the script cannot determine if the guest
+    is vulnerable or not because Firecracker does not expose the microcode
+    version to the guest.
+
+    The check in spectre_meltdown_checker is here:
+        https://github.com/speed47/spectre-meltdown-checker/blob/master/spectre-meltdown-checker.sh#L6635-L6637
+
+    Since we have a test on host and the exception in guest is not valid,
+    we add a check to ignore this exception.
+    """
+    report_guest_vulnerabilities = spectre_meltdown_reported_vulnerablities(status)
+    if global_props.instance == "m6i.metal":
```

Comment:
sorry missed this one the 1st time around... I think this would apply to c6i instances as well... 
`global_props.cpu_model == "INTEL_ICELAKE"`